modules/oadp-self-signed-certificate.adoc

```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero  describe backup  <backup-name> --details --cacert /tmp/your-cacert.txt
 ----
+
+. To fetch the logs for this specific backup, which are useful for viewing failures and warnings, including resources that could not be backed up, run the following command:
 +
+[source,terminal]
+----
+$ velero backup logs  <backup-name>  --cacert /tmp/your-cacert.txt
```

Comment:
```suggestion
$ velero backup logs  <backup_name>  --cacert /tmp/<your_cacert.txt>
```

Reply:
@anarnold97 Something like this change still needs to be applied.